src/components/pip/pip.scss

```diff
@@ -1,24 +1,23 @@
 @import '~styles/variables';
 @import '~styles/styles';
 
+$animation-duration: 0.3s;
+
 .childPlayer {
   position: absolute;
 
-  .videoContainer {
-    border-radius: 8px;
-    overflow: hidden;
-    box-shadow: 0 0 40px 0 rgba(0, 0, 0, 0.2), 0 0 4px 0 rgba(0, 0, 0, 0.3);
+  &:not(.dragging) {
+    transition: all $animation-duration ease-in;
   }
 
-  &.bottom {
-    bottom: 0;
+  &.playerHover {
+    margin-top: 50px;
   }
 
-  &.right {
-    right: 0;
-  }
-  &.left {
-    left: 0;
+  .videoContainer {
+    border-radius: 8px;
+    overflow: hidden;
```

Comment:
use maybe general css

Reply:
current styles relevant only for PIP component